### DISCUSSION TITLE
templates with syntax error (use -validate flag for further examination)

### DISCUSSION THREAD

**Discussion by aushaih** [projectdiscovery/nuclei-templates#7641] [Q&A]
can some one Please explain to me why templates I downloaded from here cant work 
![nucl1](https://github.com/projectdiscovery/nuclei-templates/assets/79006919/9350f97b-a7d4-4732-b765-b1b83e7f4da0)

for example i use CVE-2016-1000133.yaml and it dosent work but when i used the same CVE-2016-1000133.yaml from other templates collections it works 100% 

can someone help me and thanks

**Comment by DhiyaneshGeek** [comment]
Hi @aushaih Looks like you are using an outdated nuclei version

kindly update the engine and re-download the latest version of nuclei template 

```nuclei -update```

**Comment by Krosskid12** [comment]
Hi i am having the same issue even though I updated nuclei it shows same?
![image](https://github.com/projectdiscovery/nuclei-templates/assets/73221787/a7db0e6f-0bb4-43d5-a8cc-236f38cd0423)

**Comment by aushaih** [comment]
> Hi @aushaih Looks like you are using an outdated nuclei version
> 
> kindly update the engine and re-download the latest version of nuclei template
> 
> `nuclei -update`

I do update it , its v2.9.7 and for the template also updated v9.5.3 (latest)
after updated now issue appears 
(  WRN] Found 354 templates loaded with deprecated protocol syntax, update before v3 for continued support.)
all the old ones i used to find bug with not working and the now templates also not working 
al already reinstall the tools with different way  , i don't know whats the problem

**Comment by ehsandeep** [comment]
@aushaih `nuclei -reset`

**Comment by aushaih** [comment]
> @aushaih `nuclei -reset`

I've already done it but no The problem is not solved
even if i used the default /root/nuclei-templates. not working with such basic bug in http://testphp.vulnweb.com/

for example like that templates 
https://github.com/HernanRodriguez1/ScanReflectedXSS
Usually it shows 34 bug in testphp.vulnweb but now nothing 

the issues i have is 
((Found 1 templates loaded with deprecated protocol syntax, update before v3 for continued support.))

if any one have the same before and solve it 
Share the solution with us I am new in the field
thank you 

![nucl1](https://github.com/projectdiscovery/nuclei-templates/assets/79006919/b52969f5-fdad-4ca2-a95d-5fc86831ed60)

**Comment by mastercho** [comment]
I have reported this issue to PD team in Discord but they ignore me thinking its was only on me, issue is that on April they did templates rebase update and nuclei failed to delete old templates and re-downloaded rebased in top of older templates. 

To solve the issue:
You have to delete nuclei-templates folder and download it again (preferably manually to avoid further issues)

**Comment by aushaih** [comment]
> I have reported this issue to PD team in Discord but they ignore me thinking its was only on me, issue is that on April they did templates rebase update and nuclei failed to delete old templates and re-downloaded rebased in top of older templates.
> 
> To solve the issue: You have to delete nuclei-templates folder and download it again (preferably manually to avoid further issues)

i did that and not work the problem is my old templates don't work i dont want the nuclei-templates to work its useless anyway 
i want my own templates or public once more then half of them didn't work 

for some i see 
nuclei -reset or nuclei -update for me nether of them solve the issues
and even as you said "delete nuclei-templates folder and download it again" didn't solve the issues

**Comment by princechaddha** [comment]
Hey @aushaih, could you please share the output of Nuclei after updating the Nuclei engine and running nuclei -reset? 

If I understand correctly, the issue here is that the template is no longer matching on http://testphp.vulnweb.com/. Is it correct?

**Comment by ehsandeep** [comment]
@aushaih it's an issue with the template https://github.com/HernanRodriguez1/ScanReflectedXSS that is invalid and outdated.

**Comment by aushaih** [comment]
> @aushaih it's an issue with the template https://github.com/HernanRodriguez1/ScanReflectedXSS that is invalid and outdated.

this template i mention its Just an example, 
is an easy target for test we don't want to mention  a real target a real website 
even newer template showmen the same issues  
""templates loaded with deprecated protocol syntax, update before v3 for continued support""

templated like 
CVE-2023-24243
CVE-2023-24488 
CVE-2023-29489
CVE-2023-34960
the list goes on 
any public templates didn't work anymore like thousands of templates 

so is it really invalid and outdated. or its some bug cause i don't think CVE-2023-34960 or  
CVE-2023-34960 is outdated
it keep shows 
"templates loaded with deprecated protocol syntax, update before v3 for continued support"

**Reply by ehsandeep** [reply]
Read https://github.com/projectdiscovery/nuclei/discussions/3648 in detail to understand what you are seeing.

> ""templates loaded with deprecated protocol syntax, update before v3 for continued support""

This is a warning and not an error, it has nothing to do with the template not working or not loading.
